docs/source/concepts/ProjectRecovery.rst

```diff
@@ -25,11 +25,11 @@ If Mantid has crashed, then on the subsequent reboot you will be presented with
     :align: center
     :alt: alternate text
 
-You can choose to attempt a full recovery, to open a recovery script or not to attempt a recovery. Full recovery will attempt to recover all workspaces present at the time of the crash as well as additional dialogs like plots or data windows. Script mode will attempt to construct a script that contains the history of all workspaces at the time of the crash. 
+You can choose to attempt a full recovery, to open a recovery script or not to attempt a recovery. Full recovery will attempt to recover all workspaces present at the time of the crash as well as additional dialogs like plots or data windows. Script mode will attempt to construct a script that contains the history of all workspaces at the time of the crash.
 
 If full project recovery runs succesfully the scripting window will remain open in MantidPlot. It is safe to close this after a recovery.
 
-**NB** This is an early version of project recovery. We think that it is a lot better than nothing, but we know it won't always work. Known caveats are listed below. Moreover, we would sincerely appreciate feedback and input from users. Contact us at `mantid-help@mantidproject.org` 
+**NB** This is an early version of project recovery. We think that it is a lot better than nothing, but we know it won't always work. Known caveats are listed below. Moreover, we would sincerely appreciate feedback and input from users. Contact us at `mantid-help@mantidproject.org <mailto:mantid-help@mantidproject.org>`__.
```

Comment:
👍 on the email link.

May we make the second sentence more optimistic? Maybe something like `We believe the current implementation adds value, but the generated recovery script may have errors or omissions.`?